spec/oauth/oidc_spec.lua

```diff
@@ -83,21 +83,41 @@ describe('OIDC', function()
 
 
     it('verifies nbf', function()
+      stub(ngx, 'now', 1123744391)
+
       local oidc = _M.new(service)
       local access_token = jwt:sign(rsa.private, {
         header = { typ = 'JWT', alg = 'RS256', kid = 'somekid' },
         payload = {
           iss = service.oidc.issuer,
           aud = 'foobar',
-          nbf = 1,
+          sub = 'someone',
+          nbf = ngx.now() + 1,
           exp = ngx.now() + 10,
         },
       })
 
       local credentials, _, _, err = oidc:transform_credentials({ access_token = access_token })
 
-      assert.falsy(credentials)
-      assert.truthy(err)
+      assert.returns_error([['nbf' claim not valid until Thu, 11 Aug 2005 07:13:12 GMT]], credentials, err)
```

Comment:
If possible, would be better to extract that `1123744391` below into its own var and transform it here. So the relationship between this date and `ngx.now` is more obvious.

Reply:
Fixed.